src/Illuminate/Support/Faker.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Illuminate\Support;
+
+use Exception;
+use Faker\Factory;
+use Faker\Generator;
+
+class Faker extends Factory
+{
+    public Generator $faker;
+
+    public function __construct()
+    {
+        $this->faker = self::create();
+    }
+
+    /**
+     * Check the given method name.
+     * Here we can choose to use our own method,
+     * Or call a faker method.
+     *
+     * @param  string  $name
+     * @param  array  $arguments
+     * @return mixed
+     */
+    public function __call(string $name, array $arguments)
+    {
+        return match ($name) {
+            'image', 'alt' => $this->faker->$name(),
```

Comment:
Shouldn't here be  `$this->$name()` instead?

Otherwise it is equal to the `default` arm

Reply:
@rodrigopedra I think you are correcrt, code could be simplified to:

```php
    public function __call(string $name, array $arguments)
    {
        return $this->faker->$name();
    }
```

Then it would still use the custom image and alt method, and pass the other calls to faker.

Reply:
Oh sure… I missed that the magic method will only be called for missing methods… might be the late hours from yesterday :)